AdminWebsite/AdminWebsite/ClientApp/src/app/booking/participant/item/participant-item.component.spec.ts

```diff
@@ -13,8 +14,9 @@ const router = {
 
 const loggerSpy = jasmine.createSpyObj<Logger>('Logger', ['error', 'debug', 'warn']);
 let bookingServiceSpy: jasmine.SpyObj<BookingService>;
+let videoHearingsServiceSpy: jasmine.SpyObj<VideoHearingsService>;
 
-describe('ParticipantItemComponent', () => {
+fdescribe('ParticipantItemComponent', () => {
```

Comment:
remove f from this describe else only this suite will run

Reply:
Resolved